schedule/__init__.py

```diff
@@ -344,6 +356,15 @@ def at(self, time_str):
         self.at_time = datetime.time(hour, minute)
         return self
 
+    def to(self, latest):
+        """Schedule the job to run at an irregular interval.
+
+        The job's interval will vary from the value given to `every` to
+        `latest`.
```

Comment:
Might want to add a note here to make it clear that the range is inclusive on both ends due to the use of `randint()`, that is `every(A).to(B).seconds` executes every N seconds such that `A <= N <= B`.